doc/user/spec_examples.md

```diff
@@ -40,6 +40,28 @@ spec:
 
 For other topology keys, see https://kubernetes.io/docs/concepts/configuration/assign-pod-node/ .
 
+## Three member cluster with preferred anti-affinity between pods and nodes (place pods in different nodes if possible)
+
+> Note: change $cluster_name to the EtcdCluster's name.
+
+```yaml
+spec:
+  size: 3
+  pod:
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+              - key: etd_cluster
+                operator: In
```

Comment:
@rphillips small nit s/etd_cluster/etcd_cluster

Reply:
done!

Thank you.